.github/workflows/automerge.yml

```diff
@@ -1,14 +1,21 @@
-name: Auto-merge minor/patch
+name: Auto-merge
+
 on:
-  schedule:
-    - cron: "0 * * * *"
+  workflow_run:
+    types:
+      - completed
+    workflows:
+      - 'Lucky Website CI'
+
 jobs:
-  test:
-    name: Auto-merge minor and patch updates
+  auto_merge:
+    name: Auto-merge
     runs-on: ubuntu-latest
     steps:
-      - uses: koj-co/dependabot-pr-action@master
+      - if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        name: Auto-merge
+        uses: ridedott/merge-me-action@v2
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          merge-minor: false
-          merge-patch: true
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PRESET: DEPENDABOT_MINOR
```

Comment:
Just a note here:
While I agree with this change, it did change one piece of functionality, MINOR is merged now (was previously `merge-minor: false`)
https://github.com/ridedott/merge-me-action#presets

I feel it's probably fine to merge minors, and just revert if things do get get odd now and again

/cc @jwoertink